jre_emul/Classes/NSString+JavaString.m

```diff
@@ -1005,6 +1004,7 @@ + (const J2ObjcClassInfo *)__metadata {
     { "indexOfString:fromIndex:", "I", 0x1, 45, 46, -1, -1, -1, -1 },
     { "intern", "LNSString;", 0x1, -1, -1, -1, -1, -1, -1 },
     { "isEmpty", "Z", 0x1, -1, -1, -1, -1, -1, -1 },
+    { "java_compareTo:", "I", 0x1, 33, 13, -1, -1, -1, -1 },
```

Comment:
Here's the script we use to update the metadata in our native-written classes. As Tom mentions the ordering shouldn't matter, but by using the script you can be sure the metadata is correct and prevent a future diff the next time Tom or I run it.
https://github.com/google/j2objc/blob/master/jre_emul/update_metadata.sh